app/code/Magento/Sales/Setup/InstallSchema.php

```diff
@@ -2007,7 +2007,7 @@ public function install(SchemaSetupInterface $setup, ModuleContextInterface $con
         )->addColumn(
             'last_trans_id',
             \Magento\Framework\DB\Ddl\Table::TYPE_TEXT,
-            32,
+            255,
```

Comment:
This should go in an upgrade script with a version bump. Leaving this change here will only make it work for new installs and won't be reflected if anyone already installed M2 and is upgrading.

Reply:
Sure. will update at weekend.